R/derive_joined.R

```diff
@@ -371,7 +375,7 @@ derive_vars_joined <- function(dataset,
     filter_if(filter_add) %>%
     select(
       !!!by_vars,
-      !!!chr2vars(names(order)),
+      !!!extract_vars(order),
```

Comment:
I think this fixed the second part of the problem though @bundfussr or at least I'm no longer seeing the duplicates warning

Reply:
Nvm it broke a couple tests